bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -239,6 +240,27 @@ public ServerConfiguration setEntryLogFilePreAllocationEnabled(boolean enabled)
         return this;
     }
 
+    /**
+     * get ReadChannelCache expire time.
+     *
+     * @return server configuration object.
+     */
+    public long getExpireReadChannelCache() {
+        return this.getLong(EXPIRE_READ_CHANNEL_CACHE, 3600000);
+    }
+
+    /**
+     * set ReadChannelCache expire time. Default value is 1h.
+     *
+     * @param millis
+     *          expire time.
+     * @return server configuration object.
+     */
+    public ServerConfiguration setExpireReadChannelCache(long millis) {
```

Comment:
it is better to call out "time" and "ms" in the setting, like READ_CHANNEL_CACHE_EXPIRE_TIME_MS